package.json

```diff
@@ -5,23 +5,23 @@
   "author": "Qiskit Development Team",
   "license": "Apache-2.0",
   "scripts": {
-    "check": "npm run check:qiskit-bot && npm run check:patterns-index && npm run check:metadata && npm run check:spelling && npm run check:internal-links && npm run check:orphan-pages && npm run check:fmt",
```

Comment:
I moved these user-facing scripts below our dev-facing scripts `fmt`, `test`, and `typecheck`